message_definitions/v1.0/common.xml

```diff
@@ -1090,7 +1090,7 @@
         <param index="7">Empty</param>
       </entry>
       <entry value="208" name="MAV_CMD_DO_PARACHUTE" hasLocation="false" isDestination="false">
-        <description>Mission item/command to release a parachute or enable/disable auto release.</description>
+        <description>Mission item/command to release a parachute or enable/disable auto release.  Note that to release a parachute it must first be enabled, so a PARACHUTE_RELEASE must be preceeded by a PARACHUTE_ENABLE</description>
```

Comment:
@peterbarker This is not correct/consistent, based on definition of `PARACHUTE_ACTION`:
0 | [PARACHUTE_DISABLE](https://mavlink.io/en/messages/common.html#PARACHUTE_DISABLE)| Disable auto-release of parachute (i.e. release triggered by crash detectors).
-- | -- | --
1 | [PARACHUTE_ENABLE](https://mavlink.io/en/messages/common.html#PARACHUTE_ENABLE) | Enable auto-release of parachute.
2 | [PARACHUTE_RELEASE](https://mavlink.io/en/messages/common.html#PARACHUTE_RELEASE) | Release parachute and kill motors.

So `PARACHUTE_ENABLE` does not enable release as a precondition to `PARACHUTE_RELEASE` "commanded release". What it does currently is is enable./disable **auto release** as a safety mechanism.

Note I'm not opposed to this change, but you'd also need to update `PARACHUTE_ENABLE`/`PARACHUTE_DISABLE` to say something like "Disable ~~auto-release~~ of parachute, both manual and automatic (i.e. release triggered by crash detectors)."

Lastly, do all parachutes support enabling, and is the default that they are enabled or disabled? My assumption is that this depends on the flight stack. Might be worth adding that to the mix
```suggestion
        <description>Mission item/command to release a parachute or enable/disable auto release.  Note parachute release may be disabled by default: a PARACHUTE_RELEASE should be preceded by PARACHUTE_ENABLE.</description>
```